src/components/BadgeButtonList.js

```diff
@@ -1,25 +1,25 @@
 import React from "react";
 import styled from "styled-components";
-import { content } from "../api/content";
 import BadgeButton from "./BadgeButton";
 
-const BadgeButtonSection = styled.div`
+const BadgeButtonSection = styled.section`
   display: flex;
   flex-direction: row;
-  width: 352px;
+  width: 370px;
   height: auto;
+  padding-left: 8px;
   justify-content: flex-start;
   align-items: center;
   flex-flow: wrap;
 `;
 
-export default function BadgeButtonList() {
+export default function BadgeButtonList({ content }) {
   return (
     <BadgeButtonSection>
       {content.map(content => {
```

Comment:
The array with the stored information is already called "content", it might be a minor improvement to access the data with something else then "content". maybe "item" or "food" or something that implies a single item of something might make it more understandable